### PR TITLE
HW7 Microservice patterns

### PR DESCRIPTION
**Pull request by seemsod1**
- added kafka to work with email sender service
- use transactional outbox pattern

- [x] need refactoring

### CODE REVIEW

File: cmd/web/routes.go [hunk 1]
```diff
@@ -17,8 +17,14 @@ func routes() http.Handler {
...
 	mux.Use(EnableCORS)
 
-	mux.Get("/rate", handlers.Repo.Rate)
-	mux.Post("/subscribe", handlers.Repo.Subscribe)
+	mux.Route("/api", func(mux chi.Router) {
+		mux.Route("/v1", func(mux chi.Router) {
+			mux.Get("/rate", handlers.Repo.Rate)
+			mux.Post("/subscribe", handlers.Repo.Subscribe)
+			mux.Post("/unsubscribe", handlers.Repo.Unsubscribe)
+			mux.Post("/sendEmails", handlers.Repo.SendEmails)
```
Comment (vladpistun) [inline cmd/web/routes.go:25]: u need this route?

File: docker-compose.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ services:
     env_file:
       - .env
     ports:
-      - "5432:5432"
+      - "5438:5432"
```
Comment (vladpistun) [inline docker-compose.yml:23]: why?

**Review by vladpistun — APPROVED** [review]

File: cmd/web/routes.go (hunk 1 above)
Comment (seemsod1) [reply cmd/web/routes.go:25]: I used it to test a new sender with kafka, so for now not need it

File: docker-compose.yml (hunk 2 above)
Comment (seemsod1) [reply docker-compose.yml:23]: because in local I have database with "5432:5432" so I can't connect to docker in IDE with 5432

**Pull request merged**